docs/en/ingest-management/fleet/add-fleet-server-cloud.asciidoc

```diff
@@ -35,6 +35,11 @@ cover `*.cname`, not `*.*.cname`). This enables us to provide the URL for
 For more information about hosting {fleet-server} on {ece}, refer to
 {ece-ref}/ece-manage-integrations-server.html[Manage your {integrations-server}].
 
+NOTE: The TLS certificates used to secure connections between {agent} and
+{fleet-server} are managed by {ecloud}. You do not need to create a private key
+or generate certificates. To connect to {fleet-server}, {agent}s need to have
+the truststore configured on the client side.
```

Comment:
Do we need to mention the truststore part? From my understanding it's on by default, and I'm not even sure we have a way to disable it for Fleet-managed agents.

Reply:
If that's the case, we should remove this sentence. I've had Elastic Agent running on my machine for so long I honestly can't remember if I had to do anything in the beginning to configure the truststore. :-) If users need to do anything, we should expand this description because it begs the question: how do I do that? let me know how to proceed...thanks!

Reply:
@joshdover Do you want me to remove the last sentence here?

Reply:
Let's remove this. I don't think it's relevant for Fleet-managed Agents.